### DISCUSSION TITLE
No debugger available, can not send 'variables'

### DISCUSSION THREAD

**Discussion by Carlsberg789** [microsoft/vscode-discussions#721] [Extension Development QnA] [answered]
Hi,

I'm trying to write & test javascripts in Visual Studio Code. I have installed Node.js. When executing code, the Terminal itself remains empty, but the Debug Console first prints the results and then on the "last line" it prints "No debugger available, can not send 'variables'

For example, the code below

```
let array= [10,20,30,40,50,60,70,80,90]
for (let x in array) {
    console.log(x);
}
console.log(array)
```

returns:
 
0
1
2
3
4
5
6
7
8
(9) [10, 20, 30, 40, 50, 60, 70, 80, 90]
No debugger available, can not send 'variables'

The result should be just figures from 0 to 8. The last 2 lines are "errors".  How do i fix this?

**Comment by gjsjohnmurray** [comment, marked as answer]
This discussion forum is targeted at extension developers.

It doesn't sound like you are developing a VS Code extension, so I suggest you raise your issue at https://github.com/microsoft/vscode/issues after first reading the guidance at https://github.com/microsoft/vscode/wiki/Submitting-Bugs-and-Suggestions
